test/test/control/ControlTests.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package test.control;
+
+import one.profiler.test.Output;
+import one.profiler.test.Test;
+import one.profiler.test.TestProcess;
+
+public class ControlTests {
+    @Test(sh = "%testbin/start_control %f1.jfr", env = {"ASPROF_COMMAND=start,cpu,file=%f2.jfr"}, nameSuffix="fake-preload")
+    @Test(sh = "%testbin/start_control %f1.jfr", nameSuffix="normal")
```

Comment:
Two questions:
- why is this `@Test` needed?
- can/should we also check that `f2.jfr` is empty? Correct me if I'm wrong, AsyncProfiler is not `LD_PRELOADED`, `ASPROF_COMMAND` will be ignored and the file won't exist